### ISSUE TITLE
Versioning of documentation

### ISSUE THREAD

**Issue by ashwin-sureshkumar** [ReactiveX/rxjs-docs#162]
Just kicking off a discussion regarding versioning documentation for different releases of RxJS. 

Similar to angular.io, vuejs.org ..

**Comment by ashwin-sureshkumar** [comment]
cc @btroncone @ladyleet @sumitarora @kwonoj

**Comment by ladyleet** [comment]
@benlesh @kwonoj just curious if you guys think we need this? i mean... will the versions change *that* much? also, i love ember's versioning of their docs - i think it's quite impressive how they handle it. we should check that out too!

**Comment by mustafamg** [comment]
@ashwin-sureshkumar , good point.  @ladyleet, good suggestion. I go for it. ember Js versioning is more convenient with API versioning. When you change the version, it brings you the old version of the same command or keeps the current if they are the same. if the command is no longer exists, it gives you a not found page.

For Angular.io and vue js, they both redirects you to a completely new website. The reader loses the focus and needs to research for the command he is interested in.

That is why I go for ember js as it is IMHO more convenient with RxJs API documentation style.

**Comment by ladyleet** [comment]
👍

**Comment by JWO719** [comment]
I think versioning is quite important for such a huge library like rxjs. Ofc there shouldn't be that much different in each version. But you could emphazise changes/improvements for the selected version for example. Maybe it would also fit to make an amount of version selectable, for example just orientate on major release or something like that. Furthermore I think this would also offer the oppertunity to promote breaking changes at a very central point (hopefully there won't be some)

**Comment by benlesh** [comment]
FYI: I think version is *very* important, especially going forward.
